Patch-Source: https://salsa.debian.org/debian/clipit/-/blob/debian/1.4.5+git20210313-3/debian/patches/missing-prototypes.patch

Author: Andreas Beckmann <anbe@debian.org>
Description: fix building with -Werror=implicit-function-declaration

--- a/src/preferences.h
+++ b/src/preferences.h
@@ -78,6 +78,8 @@ void save_preferences();
 
 void show_preferences(gint tab);
 
+void init_history_timeout_timer();
+
 void init_purge_timer();
 
 G_END_DECLS
--- a/src/main.c
+++ b/src/main.c
@@ -46,6 +46,7 @@
 #include <ctype.h>
 #include <stdbool.h>
 #include <string.h>
+#include <sys/stat.h>
 
 #define ICON "clipit-trayicon"
 #define ICON_OFFLINE "clipit-trayicon-offline"
